govc/pool/destroy.go

```diff
@@ -39,7 +40,8 @@ func (cmd *destroy) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DatacenterFlag, ctx = flags.NewDatacenterFlag(ctx)
 	cmd.DatacenterFlag.Register(ctx, f)
 
-	f.BoolVar(&cmd.recursive, "r", false, "Remove all child resource pools recursively")
+	f.BoolVar(&cmd.recursive, "r", false, "Remove all child resource pools recursively - DEPRECATED")
+	f.BoolVar(&cmd.children, "children", false, "Remove all chilren pools")
```

Comment:
on it

2016-03-29 18:10 GMT+02:00 Doug MacEachern <notifications@github.com>:

> In govc/pool/destroy.go
> <https://github.com/vmware/govmomi/pull/482#discussion_r57751215>:
>
> > @@ -39,7 +40,8 @@ func (cmd *destroy) Register(ctx context.Context, f *flag.FlagSet) {
> >  	cmd.DatacenterFlag, ctx = flags.NewDatacenterFlag(ctx)
> >  	cmd.DatacenterFlag.Register(ctx, f)
> >
> > -	f.BoolVar(&cmd.recursive, "r", false, "Remove all child resource pools recursively")
> > +	f.BoolVar(&cmd.recursive, "r", false, "Remove all child resource pools recursively - DEPRECATED")
> > +	f.BoolVar(&cmd.children, "children", false, "Remove all chilren pools")
>
> Typo in the usage. Yes, lets remove the -r flag, very unlikely it is being
> used.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/vmware/govmomi/pull/482/files/0af347cfb75d44c68186506671152a04ed5e692e#r57751215>
>